master/getting-started/kubernetes/installation/version-differences.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Considerations between Kubernetes 1.5 and 1.6
+---
+
+There are several critical differences between Kubernetes versions 1.5 and 1.6,
+some are due to API changes and some due to the tools used to deploy Kubernetes.
```

Comment:
Are the critical differences due to the API changes and deployment tools, or are there critical differences *in* the API and the deployment tools?